tests/integration/src/xvm.rs

```diff
@@ -578,3 +615,173 @@ fn wasm_caller_reverts_if_evm_callee_reverted() {
+
+#[test]
+fn calling_wasm_from_evm_works_if_sufficient_storage_deposit_limit() {
+    init_env_logger();
```

Comment:
I don't mind, but not sure if you left this here intentionally :)

Reply:
Nice catch. I forgot to remove it after debugging.